brozzler/worker.py

```diff
@@ -247,14 +247,16 @@ def _remember_videos(self, page, ydl_spy):
         if not 'videos' in page:
             page.videos = []
         for txn in ydl_spy.transactions:
-            if (txn['response_headers'].get_content_type().startswith('video/')
+            content_type = txn['response_headers'].get_content_type()
+            if (content_type.startswith('video/')
+                    and content_type != 'video/vnd.mpeg.dash.mpd'
```

Comment:
Could you please add a comment somewhere nearby explaining why this mimetype is excluded?